src/qt/walletcontroller.cpp

```diff
@@ -164,17 +164,26 @@ WalletControllerActivity::~WalletControllerActivity()
     delete m_progress_dialog;
 }
 
-void WalletControllerActivity::showProgressDialog(const QString& label_text)
+void WalletControllerActivity::showProgressDialog(const QString& title_text, const QString& label_text)
 {
     m_progress_dialog = new QProgressDialog(m_parent_widget);
-
     m_progress_dialog->setLabelText(label_text);
-    m_progress_dialog->setRange(0, 0);
+    m_progress_dialog->setRange(0, 100);
```

Comment:
I am glad you said something - this make sense now.

On linux the busy indicator has height and contrast.
it actually conveys to the user that something is happening.

![](https://user-images.githubusercontent.com/32963518/72889766-1be9b800-3d19-11ea-8428-0909fc03d622.png)

On macOS the busy indicator is thin and the colors
do not have enough contrast - unless you really look at it
the user may not even see that anything is happening.
 
![Screen Shot 2020-01-25 at 4 09 34 AM](https://user-images.githubusercontent.com/152159/73118914-9fc5bf00-3f28-11ea-9530-1d266a4c8dd9.png)

I scoured the documentation trying to find a height adjustment for the progressBar.
There doesn't appear to be one (that is simple to implement).

As you can see - the presentations are very different between Linux and Mac.

Short of writing a custom progressDialogue/Bar - maybe an adjustment
to the colors that the progress bar is using will fix the issue.

Maybe selecting a styling that has better contrast across platforms would be better.

Thanks for saving me the time...and thanks for the free lesson on cross platform frameworks.

Reply:
The height and weight adjustment probably depends on the window manager, not on Qt.
Maybe you can try, like I said above, add a `QWidget::adjustSize()` to adjust the size manually. If you don't know how, I think it you could add `m_progress_dialog->adjustSize()` on line 175 in `walletcontroller.cpp`

Reply:
> As you can see - the presentations are very different between Linux and Mac.

Yes, widget look depends on platform. On [macOS](https://doc.qt.io/qt-5/macos-issues.html) Qt uses Apple's own APIs for doing the rendering.

Reply:
> Maybe selecting a styling that has better contrast across platforms would be better.

It could help:
https://github.com/bitcoin/bitcoin/blob/28fbe68fdcac2a06f359b1e48555a3d23015c2b7/src/qt/bitcoingui.cpp#L173-L180

Reply:
> On macOS the busy indicator is thin and the colors
> do not have enough contrast - unless you really look at it
> the user may not even see that anything is happening.

The macOS theme look-and-feel could be a subject of personal opinions, though ;)

Reply:
I'm personally not a fan of using much style sheets. It just looks very weird to all your other application and I prefer that most applications look native.

Reply:
> I'm personally not a fan of using much style sheets.

me too ;)

Reply:
I agree - I appreciate the feedback. I realize that my approach to this is wrong.
I think a better approach to this issue has to do with “accessibility” issues.
I agree that a lot of UI issues are subject to individual preferences - yet there are standards as well. On Mac there is a very useful tool to conduct “accessibility audits” for UI (Accessibility Inspector) and the GUI falls very short. I will have to research to see if there is a cross platform utility that can test against industry standards for accessibility. Thanks for the feed back.

Reply:
I believe better accessibility functionality can/should be a long term consideration to the GUI layout and design but wouldn’t require radical changes to the GUI - QT seems to have robust support for visual and other impairments.

Reply:
NACK, I think busy indicator is accessibility friendly regardless of height, contrast etc..